lib/streamlit/elements/image.py

```diff
@@ -287,6 +288,7 @@ def _ensure_image_size_and_format(
     """
     from PIL import Image
 
+    pil_image: PILImage
     pil_image = Image.open(io.BytesIO(image_data))
```

Comment:
nit: wouldn't it be possible just to do: `pil_image: PILImage = Image.open(io.BytesIO(image_data))` here?

Reply:
Thank you 😄